sys/riscv/riscv/uio_machdep.c

```diff
@@ -102,10 +107,26 @@ uiomove_fromphys(vm_page_t ma[], vm_offset_t offset, int n, struct uio *uio)
 				goto out;
 			break;
 		case UIO_SYSSPACE:
+#if __has_feature(capabilities)
```

Comment:
I copied this from MIPS, but memcpy on RISC-V is tag preserving (I changed the generic C memcpy in the kernel to copy caps in its loop, so it should preserve tags), so I think this entire condition can be removed.

Reply:
Done